### PR TITLE
Fix Bug 1132867 - Allow using mozregression with a particular pref

### PR DESCRIPTION
**Pull request by imjalpreet**
Review : @parkouss

### CODE REVIEW

**Comment by imjalpreet** [comment]
@parkouss, I have done the changes as requested by you.

File: mozregression/launchers.py [hunk 1]
```diff
@@ -110,11 +110,13 @@ def _install(self, dest):
...
         if profile:
-            profile = self.profile_class(profile=profile, addons=addons)
+            profile = self.profile_class(profile=profile, addons=addons,
+                                         preferences=preferences)
         elif len(addons):
-            profile = self.profile_class(addons=addons)
+            profile = self.profile_class(addons=addons,
+                                         preferences=preferences)
         else:
             profile = self.profile_class()
```
Comment (parkouss) [inline mozregression/launchers.py]: You missed the preferences here.

File: mozregression/main.py [hunk 2]
```diff
@@ -255,6 +265,32 @@ def bisect_nightlies(runner, logger):
     return runner.bisect_nightlies(good_date, bad_date)
 
 
+def Preferences(prefs_files, Prefs):
```
Comment (parkouss) [inline mozregression/main.py]: We usually follow the pep8 convention. Could you rename "Preferences" it to "preferences" (lowercased) ? Also "Prefs" is not good, maybe "prefs_args".

**Comment by imjalpreet** [comment]
Yeah, I have changed as per pep8 convention.

**Comment by parkouss** [comment]
Thanks! I merged this manually.

**Pull request closed without merging**